pipeline/inputs/http.md

```diff
@@ -28,7 +28,73 @@ The http input plugin allows Fluent Bit to open up an HTTP port that you can the
 
 #### How to set tag
 
-The tag for the HTTP input plugin is set by adding the tag to the end of the request URL. This tag is then used to route the event through the system. For example, in the following curl message below the tag set is `app.log`**. **If you do not set the tag `http.0` is automatically used. If you have multiple HTTP inputs then they will follow a pattern of `http.N` where N is an integer representing the input.
+The tag for the HTTP input plugin is set by adding the tag to the end of the request URL. This tag is then used to route the event through the system. For example, in the following curl message below the tag set is `app.log**. **` because the end end path is `/app_log`:
```

Comment:
Can you split up the sentences so they are on separate lines? It makes it easier to review as you can only comment on single lines - I know it wasn't before but tradition is never a good answer :)